docs/proactive_ingestion_api_documentation.md

```diff
@@ -136,14 +150,248 @@ When a Data Source creates a Data Batch, several fields must be filled out:
+   
+6. `Trace Data Transformations` - must be set to empty string. This ensures that any Data Transformations tags are cleared.
+
+7. `Relativity Attachment ID` - must be set to empty string  and mapped to `Relativity Attachment ID` field in Relativity. This field is needed for RSMF content.
+
+8. `Group Identifier` - cannot be empty. This field is used to group a parent native file and its children (e.g. an email and its attachments). Every parent native file must have a unique value. Extracted children should have the same value as its parent native file. It is suggested that a unique GUID is used for this value, and that it is different from the values used in other columns.
+
+   1. >  **IMPORTANT:** column name in the load file must be called `Group Identifier`
+
+#### Recommended
+
+1. `Trace Data Batch` – name of the batch
+2. `Trace Checkout` – must be set to empty string. This ensures Trace can
+   restart indexing and term searching for particular item in case of changes
+   to an existing document (overlay scenario)
+3. `Trace Rules` – must be set to empty string. This ensures that any Rule tags
+   are cleared.
+4. `Trace Rule Terms` – must be set to empty string. This ensures that any Rule Terms tags
+   are cleared.
+5. `Trace Terms` - must be set to empty string. This ensures that any Term tags
+   are cleared.
+6. `Trace Record Origin Identifier` - this should be an id of a record from it's data origin system. Once included, Trace will track this value across all Trace actions on a particular record. This helps to reconstruct audit trail for a particular record.
+
+![](media/7beaeffb89e2aef4285696f44c7ff423.png)
+
+### Data Batches
+
+Recommended workflow for Data Batch creation ( `Created` -\> `RetrievedFromSource` -\> `Normalized` -\> `ReadyForImport` ):
+
+1.  Create Data Batch with Status `Created` before starting data retrieval process
+    
+2.  Update Data Batch status to `RetrievedFromSource` when data is retrieved
+
+3.  If data needs to be enriched or transformed, update Data Batch status to `Normalized`
+    
+4.  Once Load File is created, update Data Batch status to `ReadyForImport`. Include Import Configuration details. At this point Trace will pick it up and ingest the data into Relativity.
+
+At any stage, you can use Metadata field on the Data Batch to capture information regarding the batch of documents as you go through the process.
+
+### Error Handling
+
+1.  For item-level errors, always populate the following two fields
+
+    1.  `Trace Has Errors` – true/false. True, if particular document has errors (e.g. audio file too big to transcribe, etc…)
+    
+2.  `Trace Error Details` – details of the individual item’s error (stack trace, retry information, etc)
+    
+2.  For batch-level error (e.g. connection failure to retrieve the data, etc...), update status of `Created` batch to `Abandoned` and populate batch `Error Details` with appropriate error.
+
+# (2) Ingest Monitored Individuals Information

```

Comment:
Can you add something to the standard Monitored Individual documentation on the https://relativitydev.github.io/relativity-trace-documentation/user_documentation#monitored-individuals that points back to this documentation. I want users to know that this exists without having to be told.